client/src/details/details.cpp

```diff
@@ -473,3 +485,42 @@ void Details::setItemsTreeModel(ItemsTreeModel *model)
     // keep the member - could be useful later
     mItemsTreeModel = model;
 }
+
+void Details::copyAddressFromGroup(QGroupBox *box)
+{
+    QMap<QString, QString> fieldNames;
+    fieldNames.insert(QStringLiteral("city"), {});
+    fieldNames.insert(QStringLiteral("country"), {});
+    fieldNames.insert(QStringLiteral("postalcode"), {});
+    fieldNames.insert(QStringLiteral("state"), {});
+    fieldNames.insert(QStringLiteral("street"), {});
+    fillAddressFielsMap(box, fieldNames);
+
+    QMap<QString, QString> address;
+    QMutableMapIterator<QString, QString> i(fieldNames);
```

Comment:
Why Mutable?

(I would personally use an stl iterator, with "auto" for the iterator type, then it all fits in one standard `for` loop, but that's minor)

Reply:
Fixed.